x-pack/legacy/plugins/actions/index.ts

```diff
@@ -42,6 +42,10 @@ export function actions(kibana: any) {
             )
             .sparse(false)
             .default([]),
+          enabledTypes: Joi.array()
+            .items(Joi.string())
+            .sparse(false)
+            .default(['*']),
```

Comment:
nit. It would be nice to use the enum here, rather than another palces specifying '*'

Reply:
added, and fixed WL as well :wink: